index.html

```diff
@@ -202,34 +321,37 @@ <h1 class="greet">
               </li>
               <li class="Css_2">
                 <img src="assets/cssLogo.png"><p>Css</p>
-              </li>
+              </li> </div> 
             </ul>
             
             <ul class="framework">
             <li >
-              <p>FrameWork</p>
+              <p>FrameWork </p>
             </li>
-            <li><img src="assets/Unionarrow.png"></li>
+            <li class="arrow1"><img src="assets/Unionarrow.png"></li>
+            <li class="separator"><img src="assets/Separator Bottom.png"></li>
             </ul>
             <ul class="skill_txt">
               <li>
                 <p>
-                Skills
+                Skills 
                 </p>
               </li>
-              <li><img src="assets/Unionarrow.png"></li>
+              <li class="arrow1"><img src="assets/Unionarrow.png"></li>
+              <li class="separator"><img src="assets/Separator Bottom.png"></li>
           </ul>
           </ul>
-      </div>
+        </div>
+      </section>
     </main>
     <footer class="contact-container">
       <div class="contact-bg">
       <div class="contacts">Contact me</div>
       <div class="contacts-txt">
         If you have an application you
-        are intrested in developing, a feature
+        are intrested in developing,<br> a feature
         that you need to build or a project that needs
-        coding . I'd love to help with it 
+        coding .<br> I'd love to help with it 
       </div>
       <form  action="https://formspree.io/f/xzbqzvgj" method="POST">
```

Comment:
Good work fixing the button 👍 But there are still few things left. Let's fix them now
- [ ] Please make sure that you remove the black border from the `input` fields and avoid using `border-radius` so that we have a design exactly given in figma template. You can do that by simply using `border: none;` for `input` to remove border form it.
- [ ] I can see that you have decreased the height of the `input` fields. Please make sure that you fix the `width` as well as suggested by previous reviewer so that we have a design according to the figma tempalte. I know you can do that 💪
- [OPTIONAL] Also, what do you think of fixing the space between each element of contact form? 🤔 Isn't it good so that we can replicate the figma template. Right now the spacing is uneven between all elements. Can we fix this? 🎯 I know you can do that

## **_Your design previously 👇_**
![image](https://user-images.githubusercontent.com/106119105/213882137-ea2842a9-3028-4b90-80e0-8cad64c02bcb.png)

## **_Your design now 👇_**
![image](https://user-images.githubusercontent.com/106119105/213882386-02ac37bd-e3be-4989-9680-4c5ba42eb809.png)

## **Figma design 👇**
![image](https://user-images.githubusercontent.com/106119105/213882125-1a9ce86d-7230-47bd-b124-2d5826ade8a8.png)